<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-05-12T14:56:40 EDT" name="Surefire suite" finished-at="2025-05-12T14:56:44 EDT" duration-ms="3275">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:runner.TestRunner@a514af7]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2025-05-12T14:56:40 EDT" name="Surefire test" finished-at="2025-05-12T14:56:44 EDT" duration-ms="3275">
      <class name="runner.TestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:runner.TestRunner@a514af7]" started-at="2025-05-12T14:56:40 EDT" name="setUpClass" finished-at="2025-05-12T14:56:41 EDT" duration-ms="903" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@6175619b]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:runner.TestRunner@a514af7]" started-at="2025-05-12T14:56:41 EDT" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2025-05-12T14:56:44 EDT" duration-ms="2204" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["User searches for the channel which is valid and invalid channel names"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Search for a channel"]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.SessionNotCreatedException">
            <message>
              <![CDATA[Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The application at 'C:\Users\mktem\eclipse-workspace\Team3_AppiumArchitects/apps/Android.Mobile.skytube.oss_55.apk' does not exist or is not accessible 
Host info: host: 'DIVYAPC', ip: '10.0.0.14'
Build info: version: '4.32.0', revision: 'd17c8aa950'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.5'
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[Capabilities {appium:app: C:\Users\mktem\eclipse-work..., appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, platformName: ANDROID}]}]
Capabilities {appium:app: C:\Users\mktem\eclipse-work..., appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, platformName: ANDROID}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The application at 'C:\Users\mktem\eclipse-workspace\Team3_AppiumArchitects/apps/Android.Mobile.skytube.oss_55.apk' does not exist or is not accessible 
Host info: host: 'DIVYAPC', ip: '10.0.0.14'
Build info: version: '4.32.0', revision: 'd17c8aa950'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.5'
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[Capabilities {appium:app: C:\Users\mktem\eclipse-work..., appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, platformName: ANDROID}]}]
Capabilities {appium:app: C:\Users\mktem\eclipse-work..., appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, platformName: ANDROID}
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:114)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:178)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:243)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:101)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:113)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)
at hooks.Hooks.setUp(Hooks.java:30)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.SessionNotCreatedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:runner.TestRunner@a514af7]" started-at="2025-05-12T14:56:41 EDT" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2025-05-12T14:56:44 EDT" duration-ms="2204" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["User searches for the channel which is valid and invalid channel names"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Search for a channel"]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.SessionNotCreatedException">
            <message>
              <![CDATA[Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The application at 'C:\Users\mktem\eclipse-workspace\Team3_AppiumArchitects/apps/Android.Mobile.skytube.oss_55.apk' does not exist or is not accessible 
Host info: host: 'DIVYAPC', ip: '10.0.0.14'
Build info: version: '4.32.0', revision: 'd17c8aa950'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.5'
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[Capabilities {appium:app: C:\Users\mktem\eclipse-work..., appium:automationName: UIAutomator2, platformName: ANDROID}]}]
Capabilities {appium:app: C:\Users\mktem\eclipse-work..., appium:automationName: UIAutomator2, platformName: ANDROID}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The application at 'C:\Users\mktem\eclipse-workspace\Team3_AppiumArchitects/apps/Android.Mobile.skytube.oss_55.apk' does not exist or is not accessible 
Host info: host: 'DIVYAPC', ip: '10.0.0.14'
Build info: version: '4.32.0', revision: 'd17c8aa950'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.5'
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[Capabilities {appium:app: C:\Users\mktem\eclipse-work..., appium:automationName: UIAutomator2, platformName: ANDROID}]}]
Capabilities {appium:app: C:\Users\mktem\eclipse-work..., appium:automationName: UIAutomator2, platformName: ANDROID}
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:114)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:178)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:243)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:101)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:113)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)
at hooks.Hooks.setUp(Hooks.java:30)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.SessionNotCreatedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:runner.TestRunner@a514af7]" started-at="2025-05-12T14:56:44 EDT" name="tearDownClass" finished-at="2025-05-12T14:56:44 EDT" duration-ms="91" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- runner.TestRunner -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
